lib/efi.py

```diff
@@ -442,6 +443,8 @@ def esl_from_auth_bytes(auth: bytes) -> bytes:
     """
     return auth[auth.index(EFI_CERT_X509_GUID):]
 
+def get_md5sum_from_auth(auth):
+    return hashlib.md5(esl_from_auth_file(auth)).hexdigest()
```

Comment:
This (and a few more funcs) are reported as moved code by `git show --color-moved --color-moved-ws=allow-indentation-change`, it could be more clear to separate code reorg from new tests